vividus-plugin-web-app/src/main/java/org/vividus/selenium/IBrowserWindowSizeManager.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2019-2022 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.vividus.selenium;
+
+import java.util.function.Consumer;
+import java.util.function.Predicate;
+
+import org.openqa.selenium.WebDriver.Window;
+
+public interface IBrowserWindowSizeManager
+{
+    void resizeBrowserWindow(BrowserWindowSize targetSize, Predicate<BrowserWindowSize> screenSizeCheck,
+                             Consumer<Window> nonConditional);
```

Comment:
the parameters are not clear at all, `screenSizeCheck` can be guessed somehow, but `nonConditional` is not clear. Please, either give clear names or add JavaDocs

Reply:
the same for the method below